swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id, condition)
+        self.fabric = fabric
+    
+    def get_category(self):
+        return "Clothing"
+    
+    def __str__(self):
+        first_part = super().__str__()
```

Comment:
👍 Nice use of `super()` to be able to call the `__str__` of the superclass, providing the first sentence of the string representation. The superclass `__str__` makes use of `get_category` to inject the correct class name, making the superclass implementation reusable by the subclasses, even though there is subclass-dependent information that needs to be embedded in the string.